src/Styles/book.css

```diff
@@ -0,0 +1,142 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.book--item {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  width: 100%;
+  height: 10.625rem;
+  margin-bottom: 0.938rem;
+  padding: 2rem 1.688rem;
+  border-radius: 4px;
+  border: 0.5px solid rgba(128, 128, 128, 0.432);
+  background-color: #fff;
+}
+
+.book--col-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.data--title {
+  width: 100%;
+  height: 1.8rem;
+  margin-top: 0.188rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+}
+
+.data--category {
+  margin: 0 4rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.data--author {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.book--actions button {
+  border: none;
+  border-right: 0.5px solid rgba(128, 128, 128, 0.432);
+  background-color: transparent;
+  margin-left: 0.938rem;
+  padding-right: 0.938rem;
+  color: #4386bf;
+  font-family: 'Roboto Slab', serif;
+  cursor: pointer;
+}
+
+.book--actions button:first-child {
+  margin-left: 0;
+}
+
+.book--actions button:last-child {
+  border: none;
+}
+
+.book--col-2 {
+  align-items: center;
+  justify-content: flex-end;
+  margin-right: 1.5rem;
+  display: flex;
+}
+
+.book--col-2 * {
+  margin-right: 1rem;
+}
+
+.loader--display {
+  width: 6rem;
+}
+
+.progress--digit {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+}
+
+.progress--title {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  color: #706e6e;
+}
+
+.book--col-3 {
+  align-self: center;
+  height: 4.375rem;
+  padding-left: 3.688rem;
+  border-left: 1px solid rgba(128, 128, 128, 0.432);
+}
+
+.chapter--title {
+  color: #706e6e;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  margin-top: -8px;
+  margin-bottom: 0.5rem;
+}
+
+.chapter--info {
+  width: 100%;
+  font-family: 'RobotoSlab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+}
+
+.chapter--btn {
+  border: none;
+  color: white;
+  background-color: #0290ff;
+  margin: 1.438rem 0;
+  padding: 0.5rem 1.5rem;
+  border-radius: 3px;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.75rem;
+  font-weight: 100;
+  letter-spacing: 1px;
+  text-align: center;
+  cursor: pointer;
+}
```

Comment:
- [ ] Many CSSS rules are repeated in your styling. Kindly ensure to refactor your styling so that there are no repeated rules.

Reply:
On it!

Reply:
Done!